### ISSUE TITLE
Include `AdditiveMonoidalCategoriesTest` in the tests of MatrixCategory/CategoryOfRows

### ISSUE THREAD

**Issue by zickgraf** [homalg-project/CAP_project#915]
@TKuh Does it make sens to call `AdditiveMonoidalCategoriesTest` in the tests of MatrixCategory/CategoryOfRows?

**Comment by TKuh** [comment]
Yes, it does. It's currently not, because [this](https://github.com/homalg-project/CAP_project/issues/886#issue-1214831681) has gotten in the way. But of course, calling the additive tests can be done without calling everything with `only_primitive_options`. I'll create a PR.
